agent/model.py

```diff
@@ -0,0 +1,271 @@
+import numpy as np
```

Comment:
Rename this file to `policy.py`. "Model" has a meaning in RL that does not apply here.